docs/integrations/azure/locations.md

```diff
@@ -68,33 +137,85 @@ integrations:
 
 If you do not specify the `organizations` field the credential will be used for all organizations for which no other credential is configured.
 
+### Authenticate against an Azure DevOps organization in a different tenant
+
+If you need to authenticate against an Azure DevOps organization in a different tenant than the service principal, you have to either:
+
+- [create a multi-tenant application in Entra ID](https://learn.microsoft.com/en-us/entra/identity-platform/single-and-multi-tenant-apps).
+- [convert the existing application to a multi-tenant application](https://learn.microsoft.com/en-gb/entra/identity-platform/howto-convert-app-to-be-multi-tenant#update-registration-to-be-multitenant).
+
 :::note Note
 
-An Azure DevOps provider is added automatically at startup for
-convenience, so you only need to list it if you want to supply a
-[personalAccessToken](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate),
-a [service principal](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/service-principal-managed-identity),
-or a [managed identity](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/service-principal-managed-identity)
+Make sure that your application requests at least one Graph API permission. This is required to be able to install the application in another tenant. The least privileged permission you can request is the [email](https://learn.microsoft.com/en-us/graph/permissions-reference#email) permission with type `Delegated`. This allows the application to read the e-mail address of the signed-in user, but without the [openid](https://learn.microsoft.com/en-us/graph/permissions-reference#openid) permission users cannot actually sign in.
```

Comment:
```suggestion
Make sure that your application requests at least one Graph API permission. This is required to be able to install the application in another tenant. The least privileged permission you can request is the [`email` permission](https://learn.microsoft.com/en-us/graph/permissions-reference#email) with type `Delegated`. This allows the application to read the e-mail address of the signed-in user, but without the [`openid` permission](https://learn.microsoft.com/en-us/graph/permissions-reference#openid) users cannot actually sign in.
```
This should address the Vale error and gives users a bit more to click on.

Reply:
Done